src/lsp/sql_parser/package.toml

```diff
@@ -68,7 +68,7 @@ version = ">=1.2"
 # package tools dependencies
 [tools]
 ppx_deriving = ">=5.2.1"
-menhir = ">=20230415"
+menhir = "20230415"
```

Comment:
ditto

Reply:
A version without a relational operator in a `package.toml` is interpreted as a lower-bound by `drom`. The `>=` is optional (but I can put it back anyways).